templates/CSI/rbd/storageclass.yaml

```diff
@@ -24,9 +25,9 @@ parameters:
     
     # The secrets have to contain Ceph admin credentials.
     csi.storage.k8s.io/provisioner-secret-name: csi-rbd-secret
-    csi.storage.k8s.io/provisioner-secret-namespace: default
+    csi.storage.k8s.io/provisioner-secret-namespace: openshift-storage
     csi.storage.k8s.io/node-publish-secret-name: csi-rbd-secret
-    csi.storage.k8s.io/node-publish-secret-namespace: default
+    csi.storage.k8s.io/node-publish-secret-namespace: openshift-storage
```

Comment:
@ebenahar in the secret.yaml files present in rook repo, the secrets are created in default namespace. It wouldn't harm to change the namespace to "openshift-storage" here.
But maybe we can confirm with dev, if there was any particular reason for creating secrets in "default" project, rather than in "rook-ceph" projects

Reply:
same

Reply:
same .. we are good as per dev